notebooks/book2/02/sub_super_gauss_plot.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    draw_all_figs(n, x)
</pre><ol><li>Probably you don't need common plot of all figs here, because in original notebook they have saved individual figures, so you should also do the same, otherwise we will not able to give foot-label (a), (b), (c), etc...</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Also, keep the figure name exactly the same as before while saving.

Reply:
Then, Should I remove this function and generate four figs individually?

Reply:
Yes